website/docs/other/production-readiness.md

```diff
@@ -0,0 +1,3 @@
+# Production Readiness
+
+Workflows4s has started a Proof of Concept validating the approach and feasibility. While its
```

Comment:
should it be something like:
`Workflows4s has started a Proof of Concept validating the approach and feasibility. While its initial stages have shown promising results, the following criteria must be met to ensure production readiness:`

Reply:
Yeah, I completely forgot I havent finished it. Thanks for spotting.